test/integration/discordactions.test.js

```diff
@@ -615,9 +616,30 @@ describe("Discord actions", function () {
     let allIds;
 
     beforeEach(async function () {
-      userData[0].roles = { archived: false };
-      userData[1].roles = { archived: false };
-      userData[2].roles = { archived: false };
+      userData[0] = {
+        ...userData[0],
+        discordId: "123456789098765432",
+        discordJoinedAt: "2023-07-31T16:57:53.894000+00:00",
+        roles: { archived: false, in_discord: true },
+      };
+      userData[1] = {
+        ...userData[1],
+        discordId: "12345678909867666",
+        discordJoinedAt: "2023-07-31T16:57:53.894000+00:00",
+        roles: { archived: false, in_discord: true },
+      };
+      userData[2] = {
+        ...userData[2],
+        discordId: "123456",
+        discordJoinedAt: "2023-07-31T16:57:53.894000+00:00",
+        roles: { archived: false, in_discord: true },
+      };
+
+      getDiscordMembers[1] = {
+        ...getDiscordMembers[1],
+        roles: ["1212121212"],
+      };
```

Comment:
Can you add this is fixture if other test is not failing

Reply:
I had to do this way as there were other tests failing